source/agora/utils/SCPPrettyPrinter.d

```diff
@@ -459,7 +459,7 @@ GCOQ...LRIJ(62,500,000), GCOQ...LRIJ(62,500,000)], enrolls: [{ utxo: 0x0000...e2
     assert(MissingSig == format("%s", scpPrettify(&env)),
```

Comment:
In this module there is a warning:

> // be warned: toVec() will point to stack-allocated memory

This warning should now be removed. Actually there is a call to `toVec()` in `env.signature = sig[].toVec();`, so the `toVec()` call should be removed too. It should be just `env.signature = pair.secret.sign(msg[]);`

Reply:
Looks like it's been addressed ? I don't see the warning in this module.